cpio/cmdline.c

```diff
@@ -70,6 +70,7 @@ static const struct option {
 	{ "grzip",			0, OPTION_GRZIP },
 	{ "help",			0, 'h' },
 	{ "insecure",			0, OPTION_INSECURE },
+	{ "dereference",		0, 'L' },
```

Comment:
The options here are in alphabetical order:  Please put 'dereference' between 'create' and 'dot'

Thank you!